scripts/protoc-swagger-openapi-gen.sh

```diff
@@ -27,7 +27,7 @@ if [ -d $temp_dir ]; then
   mv ./$temp_dir ./vendor
 fi
 # XXX We will get rid of erc20 proto files on ethermint, so I remove erc20 files on this list temporary.
-proto_dirs=$(find $tendermint_dir/proto $cosmos_sdk_dir/proto $wasm_dir/proto $ibc_dir/proto $ethermint_dir/proto $pfm/proto $xpla_dir/proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq | grep -v evmos)
+proto_dirs=$(find $tendermint_dir/proto $cosmos_sdk_dir/proto $wasm_dir/proto $ibc_dir/proto $ethermint_dir/proto $pfm/proto $xpla_dir/proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq | grep -v erc20 | grep -v precisebank)
```

Comment:
circuit, counter, epochs, protocolpool also need to remove

Reply:
[f8500dc](https://github.com/xpladev/xpla/pull/174/commits/f8500dcdfb2d09cc429e2d1fdab75b61a859a9d0)